watchOS support declaration

zolomatok opened the pull request:
Added missing watchOS 2.0 support declaration in podspec

zolomatok commented:
Until this is not resolve I cannot publish a lib dependent on Async. Please merge this or get back to me.

duemunk commented:
I'm currently on vacation with very bad internet :/

There seems to be other things than the necessary change in the PR. Can you clean it up?


> On 04 Jul 2016, at 18:41, Zoltán Matók <notifications@github.com> wrote:
> 
> Until this is not resolve I cannot publish a lib dependent on Async. Please merge this or get back to me.
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>

zolomatok commented:
Oh didn't know, no hurry, enjoy your vacation!

AFAIK there are no unnecessary changes. 
- Info.plist changed because I bumped up the version to 1.7.3
- Podspec changed because I've added the watchOS platform, bumped up the version and changed the tag name
- the pbxproj and xcscheme files changed automatically when I changed the watchOS deployment target to 2.0 (advertised in the README) instead of 2.1

I haven't created the 1.7.3 git tag, but I can do that before the merge, if you want me to. (I don't know if tags are merged in PRs)
